src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskCompleteByIdControllerTest.java

```diff
@@ -317,154 +154,62 @@ public void should_succeed_and_return_204_when_a_task_that_was_already_claimed_a
     }
 
     @Test
-    public void should_return_a_404_if_task_does_not_exist_with_completion_options_assign_and_complete_true() {
```

Comment:
Is this in IT?

Reply:
Yes
PostTaskCompleteByIdControllerFailureTest::should_return_a_404_if_task_does_not_exist